stream.go

```diff
@@ -158,21 +119,151 @@ func (s *Stream) MeasurementLatest(p Params) (<-chan *measurement.Result, error)
     return ch, nil
 }
 
+// MeasurementLatestWithChan streams the latest measurement results, as described
+// by the Params, and sends them to the supplied channel. The channel supplied should
+// not be closed, but should be reused between connections of the same measurement.
+// A channel is returned that will receive an empty struct and be closed when the connection has been closed.
+//
+// Params available are:
+//
+// "msm": int - The measurement id to get results from.
+//
+// "type": string - The measurement result type to stream.
+//
+// "sourceAddress": none - Unimplemented
+//
+// "sourcePrefix": none - Unimplemented
+//
+// "destinationAddress": none - Unimplemented
+//
+// "destinationPrefix": none - Unimplemented
+//
+// "passThroughHost": none - Unimplemented
+//
+// "passThroughPrefix": none - Unimplemented
+//
+// "sendBacklog": none - Unimplemented
+//
+// "buffering": none - Unimplemented
+func (s *Stream) MeasurementLatestWithChan(p Params, ch chan *measurement.Result) (<-chan struct{}, error) {
```

Comment:
I thought about returning the c.IsAlive func instead of a chan, but I think this is better as the caller can just put the chan in a select or block on receiving from it instead of having to call the IsAlive() func in a loop. Thoughts?